Tests/UnitTests/APIToRequestMappingTests.swift

```diff
@@ -170,6 +170,18 @@ class APIToRequestMappingTests: XCTestCase {
         XCTAssert(try element.enabled == true)
     }
 
+    func testWindowHandlePosition() throws {
```

Comment:
```suggestion
    func testWindowPosition() throws {
```
For consistency with the selenium url format.